libs/fetcher.client.js

```diff
@@ -290,7 +290,7 @@ Queue.prototype = {
          * @private
          */
         _constructGetUri: function (uri, resource, params, config) {
-            var query = [], matrix = [], id_param = config.id_param, id_val, final_uri = uri + '/resource/' + resource;
+            var query = [], matrix = [], id_param = config.id_param, id_val, final_uri = uri + '/' + resource;
```

Comment:
@lenage there is always a leading slash before the resource name so the if/else you added below isn't necessary